src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting_NetFx/DocumentFormattingTest.cs

```diff
@@ -4000,18 +4799,18 @@ private void M()
                         private void M()
                         {
                             object entries = new()
-                                {
-                                    First = 1,
-                                    Second = 2
-                                };
+                            {
+                                First = 1,
+                                Second = 2
+                            };
                         }
```

Comment:
These changes are a result of the new engine fixing https://github.com/dotnet/razor/issues/7933